tests/components/homekit_controller/test_config_flow.py

```diff
@@ -481,9 +484,9 @@ async def test_discovery_already_configured_update_csharp(hass, controller):
     discovery_info = get_device_discovery_info(device)
 
     # Set device as already paired
-    discovery_info["properties"]["sf"] = 0x00
-    discovery_info["properties"]["c#"] = 99999
-    discovery_info["properties"]["id"] = "AA:BB:CC:DD:EE:FF"
+    discovery_info[zeroconf.ATTR_PROPERTIES]["sf"] = 0x00
+    discovery_info[zeroconf.ATTR_PROPERTIES]["c#"] = 99999
+    discovery_info[zeroconf.ATTR_PROPERTIES]["id"] = "AA:BB:CC:DD:EE:FF"
```

Comment:
We should still make `id` a constant at some point

Reply:
In case someone bumps into this: the corresponding PR is #59986